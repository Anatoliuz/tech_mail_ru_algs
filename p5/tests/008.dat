(273465+(1498+238)-2349758*2365)*340