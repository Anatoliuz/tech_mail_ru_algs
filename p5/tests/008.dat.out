-1889346839460